listener/listener.go

```diff
@@ -128,6 +131,20 @@ func WithFilterDescriber(filterName string, filter servicebus.FilterDescriber) M
 	}
 }
 
+//allos listeners to control subscription details for longer lived operations.
+//If you using RetryLater you probably want this
+func WithSubscriptionDetails(lock time.Duration, maxDelivery int32) Option {
```

Comment:
this need to be ManagementOption in order to be used when New the listener

Reply:
I don't think it mattters if we set it at New or at Listen but changed to managedment option.